src/base/searchengine.h

```diff
@@ -103,8 +103,10 @@ class SearchEngine: public QObject
     void searchFailed();
     void newSearchResults(const QList<SearchResult> &results);
 
+    void pluginEnabled(const QString &name, bool enabled);
```

Comment:
It isn't, yet I think it's better to include these, also, other signals are doing so too.
Or you think otherwise?